### PR TITLE
Move inventory upload to react router

### PR DESCRIPTION
**Pull request by MariaAga**
Had to make some css changes because after I changed to `PageLayout` it looked less good. Changed the class names to fit the convention :)
Screenshot from 2020-12-22 19-09-36

### CODE REVIEW

File: webpack/ForemanInventoryUpload/Components/PageHeader/PageHeader.scss [hunk 1]
```diff
@@ -0,0 +1,7 @@
+.inventory_title {
```
Comment (ShimShtein) [inline webpack/ForemanInventoryUpload/Components/PageHeader/PageHeader.scss:1]: Do we still need it? I don't see you use it anywhere

**Review by ShimShtein — CHANGES_REQUESTED** [review]

File: webpack/ForemanInventoryUpload/Components/PageHeader/PageHeader.scss (hunk 1 above)
Comment (MariaAga) [reply webpack/ForemanInventoryUpload/Components/PageHeader/PageHeader.scss]: You're right, it's not used, deleted :tada:

**Review by ShimShtein — CHANGES_REQUESTED** [review]
the contents table got smaller, is it intentional?
After PR:
Screenshot from 2020-12-27 15-06-28
Before PR:
Screenshot from 2020-12-27 15-09-11

**Comment by MariaAga** [comment]
@ShimShtein No, but I'm changing the design in the next pr (https://github.com/theforeman/foreman_rh_cloud/pull/429), so should I fix it here?

**Comment by ShimShtein** [comment]
I'm OK with not having it fixed here, as long as we make sure it's fixed in the next one.

**Pull request merged**